### PR TITLE
corerad: 0.1.9 -> 0.2.1

### PR DESCRIPTION
**Pull request by mdlayher**
###### Motivation for this change

Version bump and minor configuration change for CoreRAD and its NixOS module test. Same ideas as my previous PR: https://github.com/NixOS/nixpkgs/pull/78034.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Tests pass:

```
router # [   84.260566] corerad[762]: 2020/02/07 03:04:14 CoreRAD v0.2.0 (BETA) starting with configuration file "/nix/store/2sj89rv38pfp9a9zp4cbaimad3039qx9-corerad.toml"
router # [   84.716619] corerad[762]: 2020/02/07 03:04:15 eth1: "prefix": ::/64 [on-link, autonomous], preferred: 4h0m0s, valid: 24h0m0s
router # [   84.784834] corerad[762]: 2020/02/07 03:04:15 eth1: "lla": source link-layer address: 52:54:00:12:01:02
router # [   84.874558] corerad[762]: 2020/02/07 03:04:15 eth1: initialized, advertising from fe80::5054:ff:fe12:102
client: waiting for the VM to finish booting
client: connected to guest root shell
client: (connecting took 0.00 seconds)
(0.12 seconds)
client: waiting for success: ping -c 1 fd00:dead:beef:dead::1
(0.18 seconds)
(94.61 seconds)
Verify SLAAC on client
client: waiting for success: rdisc6 -1 -r 10 eth1
(0.63 seconds)
client: waiting for success: ip -6 addr show dev eth1 | grep -q 'fd00:dead:beef:dead:'
client # [   90.410541] systemd-logind[819]: New seat seat0.
client # [   90.455299] systemd-logind[819]: Watching system buttons on /dev/input/event2 (Power Button)
(0.36 seconds)
client: must succeed: ip -6 addr show dev eth1
(0.08 seconds)
2 out of 2 tests succeeded
test script finished in 97.47s
cleaning up
killing client (pid 9)
killing router (pid 21)
(0.01 seconds)
/nix/store/r30aymks0ks21v7g3hfa7q583a3ga00h-vm-test-run-unnamed
```

/cc @danderson @jonringer (thanks again for the reviews!)

### CODE REVIEW

**Comment by mdlayher** [comment]
@grahamcofborg test corerad

File: pkgs/tools/networking/corerad/default.nix [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 buildGoModule rec {
   pname = "corerad";
-  version = "0.1.9";
+  version = "0.2.0";
 
   goPackagePath = "github.com/mdlayher/corerad";
```
Comment (kalbasit) [inline pkgs/tools/networking/corerad/default.nix]:
```suggestion
```

Unused by `buildGoModule`.

**Review by kalbasit — DISMISSED** [review]

**Comment by mdlayher** [comment]
@GrahamcOfBorg test corerad

**Comment by mdlayher** [comment]
@kalbasit thanks for the review! Fixed.

**Comment by mdlayher** [comment]
@GrahamcOfBorg test corerad

**Comment by mdlayher** [comment]
I made some development progress in the past few days and have just bumped to 0.2.1 with a new commit. I hope that's alright, but let me know if not.

**Comment by jonringer** [comment]
more than alright, :)

**Review by jonringer — CHANGES_REQUESTED** [review]
do you mind squashing the commits:
```
corerad: 0.1.9 -> 0.2.1
nixos/corerad: update advertisement argument
```

**Comment by mdlayher** [comment]
@GrahamcOfBorg test corerad

**Comment by mdlayher** [comment]
I made a quick git rebase mistake but appear to have fixed it up. Thanks for the reviews!

**Review by jonringer — APPROVED** [review]
diff LGTM
commits LGTM

```
https://github.com/NixOS/nixpkgs/pull/79409
1 package built:
corerad
```

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

**Pull request merged**